cross-media-measurement/src/test/kotlin/org/wfanet/measurement/integration/common/InProcessKingdomIntegrationTestBase.kt

```diff
@@ -78,7 +81,11 @@ abstract class InProcessKingdomIntegrationTestBase {
           .onEach { continuationToken = it.continuationToken }
           .map { it.globalComputation }
           .onEach { logger.info("Found GlobalComputation: $it") }
-          .toList(globalComputations)
+          .collect {
+            globalComputationsMutex.withLock {
+              globalComputations.add(it)
+            }
+          }
```

Comment:
Probably doesn't matter, but I think you can reduce mutex contention with:

```
while (true) {
  val newComputations = stub.streamWhatever().whatever().whatever().toList()
  globalComputationsMutex.withLock { globalComputations.addAll(newComputations) }
}
``

Reply:
I thought about that, but I figured we don't get enough new computations in this test method for it to matter.